code/game/machinery/constructable_frame.dm

```diff
@@ -307,3 +307,10 @@ micro-manipulator, console screen, beaker, Microlaser, matter bin, power cells.
 							/obj/item/weapon/stock_parts/capacitor = 1,
 							/obj/item/stack/cable_coil = 1,
 							/obj/item/weapon/stock_parts/console_screen = 1)
+
+/obj/item/weapon/circuitboard/machine/deep_fryer
+	name = "circuit board (Deep Fryer)"
+	build_path = /obj/machinery/deepfryer
+	origin_tech = "programming=1"
+	req_components = list(
+							/obj/item/weapon/stock_parts/micro_laser = 1)

```

Comment:
it actually does make sense for at least a couple matter bins - gotta put the oil somewhere.